internal/commands/user.go

```diff
@@ -35,3 +37,43 @@ func getSlackUserInfo(
 
 	return &user, err
 }
+
+func getUsersInConversation(
+	ctx context.Context,
+	client bot.Client,
+	logger log.Logger,
+	channelID string,
+) *[]string {
+	logger.Info(
+		ctx,
+		"command/user.getUsersInConversation",
+		log.NewValue("params", channelID),
+	)
+
+	params := slack.GetUsersInConversationParameters{
+		ChannelID: channelID,
+	}
+
+	var members []string
+
+	for {
+		list, cursor, err := client.GetUsersInConversationContext(ctx, &params)
+		if err != nil {
+			logger.Error(
+				ctx,
+				"command/user.getUsersInConversation",
+				log.NewValue("channelID", channelID),
+				log.NewValue("error", err),
+			)
+			break
+		}
```

Comment:
I think we could return this error. Maybe return here, or at the end of the function:
```
return &members, err
```